analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -163,7 +169,15 @@ class Bundler(
                 scopeDependencies = scopes.toSortedSet()
             )
 
-            val packages = gemSpecs.values.mapTo(sortedSetOf()) { getPackageFromGemspec(it) }
+            val allProjectDeps = groupedDeps.values.flatten().toSet()
+            val uniqueHelperDeps = HELPER_SCRIPT_DEPENDENCIES.filterNot { it in allProjectDeps }
+
+            val packages = gemSpecs.values.mapNotNullTo(sortedSetOf()) { gemSpec ->
+                // Bundler itself always shows up as a package because the helper script requires it, but it should be
```

Comment:
This comment assumes that the hard-coded list only contains bundler which is in fact true, but on the other hand the commit is about not hard-coding it. So, how about making the comment more generic?

Further on, would it make sense to avoid "shows up as a package"? 
However, I propose to consider dropping the comment without replacement, as it is quite redundant / obvious from the code IMO.

Reply:
I'll drop the comment altogether then.